_articles/dual-booting.md

```diff
@@ -10,100 +10,350 @@ keywords:
+
+![Windows deleting existing partitions](/images/dual-booting/windows-delete-existing-partitions.jpg)
+
+(If your flash drive is large enough, it may also show up in the list of drives; once again, identify it by its size and do not attempt to delete its partitions, since it's where the installer is stored.)
+
+You will be asked to confirm any partitions you delete:
+
+![Windows confirming a partition deletion](/images/dual-booting/windows-confirm-partition-delete.jpg)
+
+Once all of the partitions have been deleted from the soon-to-be Windows drive, select the `Unallocated Space` on that drive and click `Next`. This will allow the installer to configure the necessary partitions automatically.
+
+![Windows selecting unallocated space](/images/dual-booting/windows-select-unallocated-space.jpg)
+
+Next, wait for the Windows installation to complete. The installer will show progress as it copies and expands the OS files. The system may reboot one or more times.
+
+![Windows being installed](/images/dual-booting/windows-installing.jpg)
+
+If you see a message about checking the disk, wait and allow the installer to check its newly-created partitions.
+
+![Windows checking the disk](/images/dual-booting/windows-checking-disk.jpg)
+
+Eventually, the Windows out-of-box experience wizard will start. Select your country (it may be automatically selected if you're connected to the internet) and click `Yes`.
+
+![Windows first-time setup wizard](/images/dual-booting/windows-oobe.jpg)
+
+Follow the wizard to select your preferences:
+
+- Keyboard layout
+- Network setup, if using WiFi
+- Account details
+- Privacy settings
+- Cortana settings
+
+After completing the wizard, Windows will display a slideshow while it finishes setting up. Eventually, you will see the Windows desktop:
+
+![Windows 10 desktop](/images/dual-booting/windows-10-desktop.jpg)
+
+See [additional setup](#additional-setup-for-windows) for next steps.
 
 ---
 
-## DISCLAIMER
+## Installing on a Shared Drive (Starting with Pop!_OS)
+
+If you need to install both operating systems on a single drive (for example, if your computer only supports a single SSD), it is possible to install both OS's using a custom partition scheme. Pop!_OS's full-disk encryption is not supported with this setup.
+
+Start by [installing Pop!_OS](/articles/install-pop/), making sure to select `Don't Encrypt` during the installation process.
+
+(System76-imaged computers give you the option of whether or not to encrypt on first boot. If you chose to encrypt at first boot, or if you reinstalled and chose to encrypt, you will need to reinstall without encrypting before proceeding. If you want to dual boot and also use full-disk encryption, you will need to [use a dedicated drive](#installing-on-a-dedicated-drive) instead.)
```

Comment:
This is only true when Pop is installed and not Ubuntu so we may need to add that. The rest looks amazing.

Reply:
This article is specifically about dual booting with Pop!_OS, but I've removed the ambiguity with https://github.com/system76/docs/pull/550/commits/7461589c7b45cc819d7ebc2316f53c62b8739413 - is the new wording good?

Reply:
Looks good I just want it as clear as possible.